test/support/test_helpers.ex

```diff
@@ -4,4 +4,6 @@ defmodule CogApi.TestHelpers do
   end
 
   def get_value({:ok, value}), do: value
+  def get_value({:error, value}), do: value
```

Comment:
This seems weird to me. It's hiding the fact that you may have just gotten an error. Where is this being used? I'm curious what it looks like/if it would be better to handle this explicitly in the test case.